container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/logs_aggregator_functions/destroy_logs_aggregator.go

```diff
@@ -33,18 +33,21 @@ func DestroyLogsAggregator(ctx context.Context, dockerManager *docker_manager.Do
 		return stacktrace.Propagate(err, "An error occurred removing the logs aggregator container with ID '%v'", maybeLogsAggregatorContainerId)
 	}
 
-	maybeLogsAggregatorVolumeName, err := getLogsAggregatorVolumeName(ctx, dockerManager)
+	// We only destroy logs aggregator config volume and not data volume because we may want to restart the logs aggregator
```

Comment:
Now, that we create another volume to store vector logs disk data, we should add a way to clean it as well. This data definitely needs to persist across engine starts and stops, but we give the option to users to clean everything in a kurtosis environment using `-a` so this is a good time to clean/remove buffered data volume stored as well.